srcpkgs/cbonsai/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'cbonsai'
+pkgname=cbonsai
+version=1.3.1
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=gnu-makefile
+makedepends="ncurses-devel ncurses-libtinfo-devel pkg-config scdoc"
```

Comment:
Move `pkg-config` and `scdoc` to hostmakedepends

Reply:
Done